samples/graph-rsc/nodeJs/auth.js

```diff
@@ -0,0 +1,35 @@
+var request = require('request');
+var graph = require('@microsoft/microsoft-graph-client');
+var Q = require('q');
+
+// The auth module object.
+var auth = {};
+
+auth.getAccessToken = function () {
+    var deferred = Q.defer();
+    
+    var requestParams = {
+      grant_type: 'client_credentials',
+      client_id: process.env.ClientId,  
+      client_secret: process.env.ClientSecret,  
+      scope: 'https://graph.microsoft.com/.default'
+    };
+  
+    var url = "https://login.microsoftonline.com/"+process.env.tenantId+"/oauth2/v2.0/token";
```

Comment:
Could you please update this to fetch it from Tab Context?

Reply:
fixed.